From 21f0e4c6bc165bc403d6b3afd771f18852d3272b Mon Sep 17 00:00:00 2001
From: dseefeld <dseefeld@microsoft.com>
Date: Mon, 22 Jun 2020 12:33:58 -0400
Subject: [PATCH] Remove trailing separator for PackagePath
Trailing slash results in invalid package contents containing
a double slash which causes content to be included with an 
action of "compile" rather than "none" on usage.
See: https://github.com/NuGet/Home/issues/9705

---
 .../NuGet.Build.Tasks.Console.csproj                          | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/NuGet.Core/NuGet.Build.Tasks.Console/NuGet.Build.Tasks.Console.csproj b/src/NuGet.Core/NuGet.Build.Tasks.Console/NuGet.Build.Tasks.Console.csproj
index cff92bf71..69082f550 100644
--- a/src/NuGet.Core/NuGet.Build.Tasks.Console/NuGet.Build.Tasks.Console.csproj
+++ b/src/NuGet.Core/NuGet.Build.Tasks.Console/NuGet.Build.Tasks.Console.csproj
@@ -60,8 +60,8 @@
 
   <Target Name="PackBuildOutputs">
     <ItemGroup>
-      <TfmSpecificPackageFile Include="$(TargetPath)" PackagePath="contentFiles\any\$(TargetFramework)\" PackageCopyToOutput="true" />
-      <TfmSpecificPackageFile Include="$(ProjectRuntimeConfigFilePath)" Condition="'$(TargetFramework)' == '$(NetCoreTargetFramework)'" PackagePath="contentFiles\any\$(TargetFramework)\" PackageCopyToOutput="true" />
+      <TfmSpecificPackageFile Include="$(TargetPath)" PackagePath="contentFiles\any\$(TargetFramework)" PackageCopyToOutput="true" />
+      <TfmSpecificPackageFile Include="$(ProjectRuntimeConfigFilePath)" Condition="'$(TargetFramework)' == '$(NetCoreTargetFramework)'" PackagePath="contentFiles\any\$(TargetFramework)" PackageCopyToOutput="true" />
     </ItemGroup>
   </Target>
 
-- 
2.18.2

